modules/core/src/lib/base-attribute.js

```diff
@@ -110,15 +110,15 @@ export default class BaseAttribute {
     return null;
   }
 
-  _createBuffer(opts) {
+  _createBuffer() {
     // Move accessor fields to accessor object
-    const props = Object.assign({}, opts, {
+    const props = {
       id: this.id,
       target: this.target,
       accessor: {
         type: this.type
       }
-    });
+    };
     if (Number.isFinite(props.divisor)) {
```

Comment:
Remove

Reply:
remove passing `divisor` and `size` to `Buffer` constructor?

Reply:
You are assigning them to `props.accessor` which we are no longer passing.